nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/syslog/Syslog5424Reader.java

```diff
@@ -59,18 +59,31 @@
 @CapabilityDescription("Provides a mechanism for reading RFC 5424 compliant Syslog data, such as log files, and structuring the data" +
         " so that it can be processed.")
 public class Syslog5424Reader extends SchemaRegistryService implements RecordReaderFactory {
+
     public static final String RFC_5424_SCHEMA_NAME = "default-5424-schema";
     static final AllowableValue RFC_5424_SCHEMA = new AllowableValue(RFC_5424_SCHEMA_NAME, "Use RFC 5424 Schema",
             "The schema will be the default schema per RFC 5424.");
+
+    static final String RAW_MESSAGE_NAME = "_raw";
+
     public static final PropertyDescriptor CHARSET = new PropertyDescriptor.Builder()
             .name("Character Set")
             .description("Specifies which character set of the Syslog messages")
             .required(true)
             .defaultValue("UTF-8")
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
+    public static final PropertyDescriptor ADD_RAW = new PropertyDescriptor.Builder()
```

Comment:
This property isn't added to the list of supported properties, so although it can be set for unit tests, it can't be set from the UI

Reply:
doh...